packages/mdc-tab/README.md

```diff
@@ -43,9 +43,106 @@ npm install --save @material/tab
   <div class="mdc-tab__content">
-    <span class="mdc-tab__icon">heart</div>
     <span class="mdc-tab__text-label">Favorites</div>
   </div>
+  <span class="mdc-tab__indicator"></span>
+  <span class="mdc-tab__ripple"></span>
+</button>
+```
+
+#### Text Label and Icon
+
+You can use an icon in place of or with a text label.
+
+##### Text Label
+
+```html
+<button class="mdc-tab" role="tab" aria-selected="false">
+  <div class="mdc-tab__content">
+    <span class="mdc-tab__text-label">Favorites</div>
+  </div>
+  <span class="mdc-tab__indicator"></span>
+  <span class="mdc-tab__ripple"></span>
+</button>
+```
+
+##### Icon
+
+```html
+<button class="mdc-tab" role="tab" aria-selected="false">
+  <div class="mdc-tab__content">
+    <span class="mdc-tab__icon material-icons">favorite</div>
+  </div>
+  <span class="mdc-tab__indicator"></span>
+  <span class="mdc-tab__ripple"></span>
+</button>
+```
+
+##### Text Label and Icon
+
+```html
+<button class="mdc-tab" role="tab" aria-selected="false">
+  <div class="mdc-tab__content">
+    <span class="mdc-tab__icon material-icons">favorite</div>
+    <span class="mdc-tab__text-label">Favorites</div>
+  </div>
+  <span class="mdc-tab__indicator"></span>
+  <span class="mdc-tab__ripple"></span>
+</button>
+```
+
+##### Text Label and Icon on Two Lines
+
+```html
+<button class="mdc-tab mdc-tab--two-lines" role="tab" aria-selected="false">
+  <div class="mdc-tab__content">
+    <span class="mdc-tab__icon material-icons">favorite</div>
+    <span class="mdc-tab__text-label">Favorites</div>
+  </div>
+  <span class="mdc-tab__indicator"></span>
+  <span class="mdc-tab__ripple"></span>
+</button>
+```
+
+#### Indicator Customization
+
+The tab indicator can be customized via dimensions and style. It can appear as a bar matching the width of the full tab *or* a bar matching the width of the text label. It can also be customized to appear as an icon.
+
+##### Full Tab Width Indicator
+
+```html
+<button class="mdc-tab" role="tab" aria-selected="false">
+  <div class="mdc-tab__content">
+    <span class="mdc-tab__text-label">Favorites</div>
+  </div>
+  <span class="mdc-tab__indicator"></span>
+  <span class="mdc-tab__ripple"></span>
+</button>
+```
+
+##### Text Label Width Indicator
+
+To make the indicator match the width of the text label, just move the indicator element inside the `mdc-tab__content` element.
+
+```html
+<button class="mdc-tab" role="tab" aria-selected="false">
+  <div class="mdc-tab__content">
+    <span class="mdc-tab__text-label">Favorites</div>
+    <span class="mdc-tab__indicator"></span>
+  </div>
+  <span class="mdc-tab__ripple"></span>
+</button>
+```
+
+##### Icon Indicator
+
+```html
+<button class="mdc-tab" role="tab" aria-selected="false">
+  <div class="mdc-tab__content">
+    <span class="mdc-tab__text-label">Favorites</div>
+  </div>
+  <span class="mdc-tab__indicator mdc-tab__indicator--icon material-icons">change_history</span>
```

Comment:
I think we generally try and leave some information about how to setup material icons font too